resources/content/schema/base/externalHandler.json

```diff
@@ -14,6 +14,11 @@
             "required" : true,
             "includeInList" : false
         },
+        "externalHandlerProcessConfigs" : {
```

Comment:
Im just talking about the property on externalHandler. The name of the type is fine and can stay externalHandlerProcessConfigs.